plugins/kernel_v2/config/vm.rb

```diff
@@ -455,7 +455,7 @@ def finalize!
         @base_mac = nil if @base_mac == UNSET_VALUE
         @base_address = nil if @base_address == UNSET_VALUE
         @boot_timeout = 300 if @boot_timeout == UNSET_VALUE
-        @box = nil if @box == UNSET_VALUE
+        @box = nil if @box == UNSET_VALUE || @box.to_s.empty?
```

Comment:
Instead, we should check the value of `@box` inside the `#validate` method, and then we can give the user a proper error if they've set the value to an empty string.